src/display/editor/ink.js

```diff
@@ -407,11 +396,32 @@ class InkEditor extends AnnotationEditor {
    */
   #draw(x, y) {
     const [lastX, lastY] = this.currentPath.at(-1);
-    if (x === lastX && y === lastY) {
+    if (this.currentPath.length > 1 && x === lastX && y === lastY) {
       return;
     }
-    this.currentPath.push([x, y]);
+    const currentPath = this.currentPath;
+    let path2d = this.#currentPath2D;
```

Comment:
Nit: In the existing code we're using `path2D`, note the capitalization, so can we perhaps use a consistent format through this file to prevent surprises/issues later on?